lib/cdo/aws/cloud_formation.rb

```diff
@@ -309,11 +309,8 @@ def render_template(dry_run: false)
           availability_zones: AVAILABILITY_ZONES,
           azs: azs,
           s3_bucket: S3_BUCKET,
-          file: method(:file),
-          js: method(:js),
-          erb: method(:erb),
```

Comment:
These `method(:fn)` variables are no longer needed due to previous changes in how the binding is attached for ERB processing. This simplifies uses of these methods to e.g., `file(params)` instead of `file.call(params)`.